crates/swc_fast_ts_strip/src/lib.rs

```diff
@@ -1062,28 +1121,34 @@ impl Visit for TsStrip {
 
     fn visit_ts_enum_decl(&mut self, e: &TsEnumDecl) {
         HANDLER.with(|handler| {
-            handler.span_err(
-                e.span,
-                "TypeScript enum is not supported in strip-only mode",
-            );
+            handler
+                .struct_span_err(
+                    e.span,
+                    "TypeScript enum is not supported in strip-only mode",
+                )
+                .emit();
         });
     }
 
     fn visit_ts_module_decl(&mut self, n: &TsModuleDecl) {
         HANDLER.with(|handler| {
-            handler.span_err(
-                n.span(),
-                "TypeScript namespace declaration is not supported in strip-only mode",
-            );
+            handler
+                .struct_span_err(
+                    n.span(),
+                    "TypeScript namespace declaration is not supported in strip-only mode",
+                )
+                .emit();
         });
     }
 
     fn visit_ts_namespace_decl(&mut self, n: &TsNamespaceDecl) {
         HANDLER.with(|handler| {
-            handler.span_err(
-                n.span(),
-                "TypeScript module declaration is not supported in strip-only mode",
-            );
+            handler
+                .struct_span_err(
+                    n.span(),
+                    "TypeScript module declaration is not supported in strip-only mode",
```

Comment:
I don't see any test fixture for this case. And I'm confused why this is needed.

Reply:
This whole visit still seems redundant given that it does not appear in any of the test fixtures.

But that is a pre-existing condition, so I am approving anyway.